draft-ietf-ppm-dap.md

```diff
@@ -1523,39 +1519,34 @@ enum {
 } ExtensionType;
 ~~~
 
-Field "extension_type" indicates the type of extension, and "extension_data"
-contains information specific to the extension.
-
-Extensions are mandatory-to-implement: If an Aggregator receives a report
-containing an extension it does not recognize, then it MUST reject the report.
-(See {{input-share-validation}} for details.)
+Field `extension_type` indicates the type of extension, and `extension_data`
+contains the opaque encoding of the extension.
 
 ## Verifying and Aggregating Reports {#aggregate-flow}
 
-Once a set of Clients have uploaded their reports to the Leader, the Leader can
+Once some Clients have uploaded their reports to the Leader, the Leader can
 begin the process of validating and aggregating them with the Helper. To enable
-the system to handle large batches of reports, this process can be parallelized
-across many "aggregation jobs" in which small subsets of the reports are
-processed independently. Each aggregation job is associated with exactly one DAP
-task, but a task can have many aggregation jobs.
+the system to handle large batches of reports, this process is parallelized
+across many "aggregation jobs" in which subsets of the reports are processed
+independently. Each aggregation job is associated with a single task, but a task
+can have many aggregation jobs.
 
-The primary objective of an aggregation job is to run the VDAF preparation
-process described in {{!VDAF, Section 5.2}} for each report in the job.
-Preparation has two purposes:
+An aggregation job runs the VDAF preparation process described in {{!VDAF,
+Section 5.2}} for each report in the job. Preparation has two purposes:
 
 1. To "refine" the input shares into "output shares" that have the desired
    aggregatable form. For some VDAFs, like Prio3, the mapping from input to
    output shares is a fixed operation depending only on the input share, but in
    general the mapping involves an aggregation parameter chosen by the
    Collector.
 
-1. To verify that the output shares, when combined, correspond to a valid,
-   refined measurement, where validity is determined by the VDAF itself. For
-   example, the Prio3Sum variant of Prio3 ({{Section 7.4.2 of !VDAF}}) requires
-   that the output shares sum up to an integer in a specific range, while the
-   Prio3Histogram variant ({{Section 7.4.4 of !VDAF}}) proves that output shares
-   sum up to a one-hot vector representing a contribution to a single bucket of
-   the histogram.
+1. To verify that each pair of output shares, when combined, corresponds to a
+   valid, refined measurement, where validity is determined by the VDAF itself.
+   For example, the Prio3Sum variant of Prio3 ({{Section 7.4.2 of !VDAF}})
+   proves that the output shares sum up to an integer in a specific range, while
+   the Prio3Histogram variant ({{Section 7.4.4 of !VDAF}}) proves that output
+   shares sum up to a one-hot vector representing a contribution to a single
+   bucket of the histogram.
```

Comment:
Clarity: output shares are combined pairwise. Actually, this is probably still confusing, because the output shares never actually get combined in DAP. We need an "if" in here.